android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -2301,7 +2301,10 @@ private void toggleTrackRecordingPP()
         && mPlacePageViewModel.getMapObject().getValue().isTrackRecording())
       mPlacePageViewModel.setMapObject(null);
     else
-      mPlacePageViewModel.setMapObject(new TrackRecording());
+    {
+      String title = "0" + getString(app.organicmaps.sdk.R.string.m) + " • 0" + getString(R.string.min);
```

Comment:
After a second thought, let's put this string into translations too like "%1$s • %2$s" and a comment (%1$s is the time of how long the track is recorded, %2$s is the recorded track length).

Note that iOS shows time and then length, let's use the same order here. RTL languages then may swap it for example.